spec/definitions.yaml

```
@@ -89,6 +89,11 @@ definitions:
               default:
                 description: Default number of availability zones for a cluster.
                 type: integer
+              zones:
+                type: array
+                description: The distinct availability zones available in the installation's region.
```

```suggestion
                description: Availability zone options for tenant cluster.
```
How about this? I'd maybe drop the `installation` from there since it's actually for the tenant cluster and I can imagine that in some corner case we could even restrict by certain BYOC account. I don't know - in all cases it still applies to tenant cluster account, not the g8s control plane / installation, so that's why I'm slightly picky here.

---

Changing to `The availability zones available for in the installation's region for use with tenant clusters.`

---

Maybe zap first `for`, but otherwise I'm good with that.